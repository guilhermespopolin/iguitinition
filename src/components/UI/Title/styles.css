.title {
  text-align: center;
  font-size: 4rem;
  color: var(--primary);
}