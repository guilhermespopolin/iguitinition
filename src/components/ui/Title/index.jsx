import styled from 'styled-components'

const Title = styled.h1`
  text-align: center;
  font-size: 4rem;
  color: ${({ theme }) => theme.colors.primary};
`

export default Title
