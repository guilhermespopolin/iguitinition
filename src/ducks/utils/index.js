import { keyBy } from 'lodash'

export function normalizePayload(payload) {
  return keyBy(payload, 'id')
}

export default {
  normalizePayload,
}
