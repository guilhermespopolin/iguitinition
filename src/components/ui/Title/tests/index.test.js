import React from 'react'
import { render } from '__tests__/utils'

import Title from '..'

describe('<Title />', () => {
  it('should render properly', () => {
    const { getByText } = render(<Title>Sample Title</Title>)
    const titleNode = getByText('Sample Title')

    expect(titleNode).toMatchSnapshot()
  })
})

