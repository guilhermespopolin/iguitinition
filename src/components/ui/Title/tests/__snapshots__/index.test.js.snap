// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render properly 1`] = `
.c0 {
  text-align: center;
  font-size: 4rem;
  color: #0a30ad;
}

<h1
  class="c0"
>
  Sample Title
</h1>
`;
